content/blog/rn-bridging-native-modules/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Bridging the gap between Swift & React Native
+date: "2020-01-02T11:00:05.169Z"
+description: A guide on how to bridge and use WebViews built with Swift in React Native (because someone had to do it)
+---
+
+# Introduction
+
+React Native is great in many ways, primarily for offering a way to build apps for both iOS and Android using the same codebase while offering plenty of tools and functionality for both platforms. However, there are rare cases where you might need some functionality that is only available through a specific platform's API. To access such functionality you need to write native modules using either Swift+Objective-C for iOS, or Java for Android, and bridge them over to React Native.  
+React Native's documentation on how to do this is in all honesty somewhat lacking at times, and at best it's pretty scattered, so a React Native developer who is new to working with native modules might find themselves spending hours piecing together bits of information from a multitude of different articles and blog posts to get even the simplest of native module to work.  
+
+This guide covers how to bridge a native iOS WebView written in Swift for usage in React Native, and also how to pass data between them. While the example used in the guide will focus on how to bridge a WebView, the same principle can be used for virtually any UIView in Swift.
+
+# Getting started
```

Comment:
If it fits here, maybe explains that this post will have a focus on bridging from Swift. Maybe also explain why you start out with Swift.